Runtime/Scripts/Rooms/Participants/Factory/IParticipantFactory.cs

```diff
@@ -27,11 +28,19 @@ FfiHandle handle
         )
         {
             var participant = factory.NewParticipant(info, room, handle, Origin.Remote);
-                foreach (var pubInfo in publications ?? Array.Empty<OwnedTrackPublication>())
-                {
-                    var publication = ITrackPublicationFactory.Default.NewTrackPublication(pubInfo.Info!);
-                    participant.AddTrack(publication);
-                }
+            foreach (var pubInfo in publications ?? Array.Empty<OwnedTrackPublication>())
+            {
+                var publication = ITrackPublicationFactory.Default.NewTrackPublication(pubInfo.Info!);
+                
+                var trackHandle = IFfiHandleFactory.Default.NewFfiHandle((IntPtr)pubInfo.Handle.Id);
+
+                var track = room.MyTracksFactory.NewTrack(trackHandle, TracksFactory.FromTPI2TI(pubInfo.Info), room, participant);
```

Comment:
I don't think we want to create tracks. 

I think instead we want to change SetSubscribedForRemote's function to not reference Track.Handle.  Instead it should reference  Publication's handle. 

So in IParticipantFactory it could be this:
var publication = ITrackPublicationFactory.Default.NewTrackPublication(pubInfo.Handle, pubInfo.Info!);

and in Room's TrackPublished event it could be this:
var publication = trackPublicationFactory.NewTrackPublication(e.TrackPublished.Publication!.Handle, e.TrackPublished.Publication!.Info!);

Then we don't need to be creating tracks on the client side if that works